dbptk-modules/dbptk-filter-inventory/src/main/java/com/databasepreservation/modules/inventory/InventoryFilterFactory.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE file at the root of the source
+ * tree and available online at
+ *
+ * https://github.com/keeps/db-preservation-toolkit
+ */
+package com.databasepreservation.modules.inventory;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.databasepreservation.model.exception.ModuleException;
+import com.databasepreservation.model.modules.filters.DatabaseFilterFactory;
+import com.databasepreservation.model.modules.filters.DatabaseFilterModule;
+import com.databasepreservation.model.modules.filters.ExecutionOrder;
+import com.databasepreservation.model.parameters.Parameter;
+import com.databasepreservation.model.parameters.Parameters;
+import com.databasepreservation.model.reporters.Reporter;
+import com.databasepreservation.utils.ModuleUtils;
+
+public class InventoryFilterFactory implements DatabaseFilterFactory {
+  public static final String PARAMETER_PREFIX = "prefix";
+  public static final String PARAMETER_DIR_PATH = "dir-path";
+  public static final String PARAMETER_PRINT_HEADER = "print-header";
+  public static final String PARAMETER_SEPARATOR = "separator";
+
+
+  private static final Parameter prefix = new Parameter().shortName("pr").longName(PARAMETER_PREFIX)
+    .description("Prefix to append to inventory files").hasArgument(true).setOptionalArgument(false).required(false);
+
+  private static final Parameter dirPath = new Parameter().shortName("dp").longName(PARAMETER_DIR_PATH)
+          .description("Dir path to save inventory files").hasArgument(true).setOptionalArgument(false).required(false);
+
+  private static final Parameter printHeader = new Parameter().shortName("ph")
+          .longName(PARAMETER_PRINT_HEADER)
+          .description("The message to indicate if the header should be printed or not (Default: true)")
+          .hasArgument(false).setOptionalArgument(false).required(false).valueIfSet("false").valueIfNotSet("true");
```

Comment:
The header will be printed by default, if this property is set the header will not be printed. Rename to disable-print-header. Value if set = true